loaders/src/glTF/babylon.glTFFileLoader.ts

```diff
@@ -592,6 +596,24 @@ module BABYLON {
                 indexCounts.push(tempVertexData.indices.length);
             }
 
+            //bitangent aka binormal
+            if (tempVertexData.tangents !== undefined && tempVertexData.tangents !== null) {
+                var bitangents: number[] = [];
+                var tindex = 0;
+                for (var i = 0; i < tempVertexData.normals.length; i += 3) {
+                    var n = Vector3.FromArray(tempVertexData.normals, i);
+                    var t = Vector4.FromArray(tempVertexData.tangents, tindex);
+                    //bitangent cross product of normal and tangent
+                    var bitangent = Vector3.Cross(n, t.toVector3().scale((t.w < 0) ? -1 : 1)).asArray();

```

Comment:
This really should happen in the shader. I realize you are trying to support TBN in the shader, but I think maybe you should support 3 modes in the shader (normal only, normal + tangent4, normal + bitangent + tangent). Feel free to not support normal + bitangnet  + tangent just yet since we don't need it.